test/chpldoc/compflags/author/withoutAuthor/CLEANFILES

```diff
@@ -0,0 +1 @@
+docs
```

Comment:
Whoops, missing newline at end of file here

Reply:
Done